Automattic/studio
# Studio: Images arent loaded in assistant responses

Fixes https://github.com/Automattic/dotcom-forge/issues/7721

## Proposed Changes

This PR fixes a bug, where when the AI Assistant returns some images, those are not displayed.

## Testing Instructions



1. Open Studio
2. Create a site
3. Navigate to the assistant tab
4. Ask a question that probably will return an image.  Mine was "How can I add a new user"
5. Ensure that the images are displayed as expected

![2024-06-21T11:11:42,039596244+03:00](https://github.com/Automattic/studio/assets/497103/dc129797-1d7d-442a-a642-68553354f45d)
![2024-06-21T11:11:52,566657908+03:00](https://github.com/Automattic/studio/assets/497103/249489b3-f6be-4bb9-bb4f-ff443f43c9f5)


## Pre-merge Checklist
